charts/helm-dep-installer/templates/namespace.yaml

```diff
@@ -1,10 +1,10 @@
-{{- if and .Values.helmChart.createNamespace (ne .Values.helmChart.namespace "") }}
-apiVersion: v1
-kind: Namespace
-metadata:
-  annotations:
-    "helm.sh/hook": {{ .Values.hook.type | default "pre-install" | quote }}
-    "helm.sh/hook-weight": {{ include "generic-dep-installer.crb-hook-value" . }}
-    "helm.sh/hook-delete-policy": "before-hook-creation,hook-failed"
-  name: {{ .Values.helmChart.namespace }}
-{{- end }}
+# {{- if and .Values.helmChart.createNamespace (ne .Values.helmChart.namespace "") }}
```

Comment:
@shydefoo  shall i delete this file then?

Reply:
yep we should remove the file in that case since we no longer need it